packages/analytics-browser/src/plugins/file-download-tracking.ts

```diff
@@ -0,0 +1,77 @@
+import { BrowserClient, PluginType, Event, EnrichmentPlugin } from '@amplitude/analytics-types';
+import { BrowserConfig } from '../config';
+
+const FILE_DOWNLOAD_EVENT = 'file_download';
+
+export const fileDownloadTracking = (): EnrichmentPlugin => {
+  const name = 'fileDownloadTracking';
+  const type = PluginType.ENRICHMENT;
```

Comment:
Do we need this to be an enrichment type, or before type is fine?

Reply:
Logically, it could be either since these two type only differs when modifying the payload